docs/source/examples.rst

```diff
@@ -4,6 +4,150 @@
 Examples
 ########
 
+.. _gym_integration:
+
+**************************
+Integration with Gymnasium
+**************************
+
+`OpenAI Gymnasium <https://gymnasium.farama.org/>`_, also known as Gym, is a popular toolkit for developing and comparing reinforcement learning algorithms by provides a standardized interface for environments. Gym offers a variety of environments, from simple classic control tasks like balancing a pole, which is described below in detail, to complex games like Atari and MuJoCo. It even supports creating custom environments, making it a versatile tool for all things reinforcement learning research.
```

Comment:
1. "also known" -> "formerly known"
2. I think we should talk about environments directly and avoid the term "RL algorithms" to avoid confusion (like reviewers who wondered what the difference is between Reinforced-lib and gymnasium).

Reply:
Will change these two points. Thanks